app/views/cooking_sessions/show.html.erb

```diff
@@ -1,9 +1,14 @@
-<h1>Cooking Sessions page</h1>
-
-<!-- app/views/reviews/new.html.erb -->
-<p>Meal being cooked: <%= @cooking_session.meal.name %></p>
-<p>Meal description: <%= @cooking_session.meal.description %></p>
-<p>Max Portions: <%= @cooking_session.max_portions  %></p>
-<p> Start At <%= @cooking_session.start_at %></p>
-<p> End At <%= @cooking_session.end_at %></p>
-
+<%# POST cooking_sessions/:cs_id/reservations %>
+<div class ="container">
+  <h1>Cooking Sessions page</h1>
+  <%= simple_form_for [@cooking_session, @reservation] do |f| %>
+    <h2 class="border-bottom">Book now!</h2>
+    <p>Meal being cooked: <%= @cooking_session.meal.name %></p>
+    <p>Meal description: <%= @cooking_session.meal.description %></p>
+    <p>Max Portions: <%= @cooking_session.max_portions  %></p>
+    <p> Booking date <%= @cooking_session.start_at %></p>
```

Comment:
I think you can delete this line as cooking_session.start_at time is not really a booking time or date